tests/jballerina-unit-test/src/test/resources/test-src/runtime/api/values/modules/xml_values/xml_values.bal

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```
The "All Rights Reserved" part needs to be removed as per the new license header for opensource codes.